lib/Echidna/Campaign.hs

```diff
@@ -219,13 +219,18 @@ execTxOptC :: (MonadState x m, Has Campaign x, Has VM x, MonadThrow m) => Tx ->
 execTxOptC t = do
   og  <- hasLens . coverage <<.= mempty
   res <- execTxWith vmExcept (usingCoverage $ pointCoverage (hasLens . coverage)) t
-  hasLens . coverage %= unionWith union og
+  let vmr = isRevert $ fst res
+  -- Update the coverage map with the proper binary according to the vm result 
+  hasLens . coverage %= mapWithKey (\ _ s -> DS.map (\(i,_) -> (i, vmr)) s)
```

Comment:
i don't think constantly overwriting is the correct plan here. maybe `modifying _2 (vmr ||)` so revert sets it forever